src/nvim/os/fs.c

```diff
@@ -192,13 +192,15 @@ int os_open(const char* path, int flags, int mode)
 
 /// Get stat information for a file.
 ///
-/// @return OK on success, FAIL if a failure occurred.
+/// @return `true` on success, `false otherwise.
 static bool os_stat(const char *name, uv_stat_t *statbuf)
   FUNC_ATTR_NONNULL_ALL
 {
   uv_fs_t request;
```

Comment:
You mean adding the `if`? No, that didn't fix it, which is why I think something else must be wrong (wrong `true` returned/access despite `false` returned?)